tcf_website/templates/profile/delete_confirm_modal.html

```diff
@@ -0,0 +1,25 @@
+<!-- Delete Confirmation Modal -->
+<div class="modal fade" id="deleteProfileModal{{user.id}}" tabindex="-1" role="dialog" aria-labelledby="deleteModalLabel" aria-hidden="true">
```

Comment:
Just wondering -does it particularly matter if the ID of this element has the user ID instead of just being called `deleteProfileModal`? I see it's used as the data target for the modal in `profile.html`, but I can't see anything that would suggest it needs to be unique per user.

Reply:
good point seems redundant i'll make that change